remoter-client/src/test_fn.r

```diff
@@ -0,0 +1,21 @@
+some_other_stuff <- function(a) {
+    a + 5
+}
+
+task <- function(input_data, input_config) {
+    config <- input_config
+
+    config$limit <- 202
```

Comment:
what is this limit for? Why testing with 202 is a good choice?

Reply:
There is no point to it, it was just to see that R's copy-on-write works properly. I can just annotate this entire function as being just a dummy and it only serves as an example.